File: client-desktop/NexusERP/Models/UretimKaydi.cs
using System;

namespace NexusERP.Models
{
    public class UretimKaydi
    {
        public int UretimID { get; set; }
        public int UrunID { get; set; }
        public int MakineID { get; set; }
        public int PersonelID { get; set; }
        public int UretimAdedi { get; set; }
        public DateTime UretimTarihi { get; set; }
        public string Aciklama { get; set; }
    }
}

File: backend/src/main/java/com/tanerkaynar/nexuserp/model/UretimKaydi.java
package com.tanerkaynar.nexuserp.model;

import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@Entity
@Table(name = "uretimkayitlari")
public class UretimKaydi {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "uretimid")
    private Integer uretimid;

    @ManyToOne
    @JoinColumn(name = "urunid", nullable = false)
    private Urun urun;

    @ManyToOne
    @JoinColumn(name = "makineid", nullable = false)
    private Makine makine;

    @ManyToOne
    @JoinColumn(name = "personelid", nullable = false)
    private Personel personel;

    @Column(name = "uretimadedi", nullable = false)
    private Integer uretimadedi = 0;

    @Column(name = "uretimtarihi", insertable = false, updatable = false)
    private LocalDateTime uretimtarihi;

    @Column(name = "aciklama", columnDefinition = "TEXT")
    private String aciklama;
}